### PR TITLE
add contributors list

### PR DESCRIPTION
**Pull request by animeshsingh**

### CODE REVIEW

File: _posts/2021-03-08-kfserving-0.5.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ KFServing 0.5 also introduces an optional model agent for request/response loggi
 
 -   [<u>TorchServe integration</u>](https://twitter.com/PyTorch/status/1346466660744511492): TorchServe now is used as implementation for KFServing PyTorch model server, it also enables model explanability with [<u>Captum</u>](https://captum.ai/), see TorchServe examples [<u>here</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/torchserve).
 
--   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>bert</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>torchscript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
+-   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>BERT</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>torchscript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
```
Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:32]:
```suggestion
-   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>BERT</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>TorchScript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
```

File: _posts/2021-03-08-kfserving-0.5.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ KFServing 0.5 also introduces an optional model agent for request/response loggi
...
```
Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:33]: missing the link for XGBoost

File: _posts/2021-03-08-kfserving-0.5.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ KFServing 0.5 also introduces an optional model agent for request/response loggi
...
 
 -   Tensorflow <u>gRPC support</u>.
```
Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:34]:
```suggestion
-   Tensorflow [<u>gRPC support</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/tensorflow#create-the-inferenceservice-with-grpc).
```

File: _posts/2021-03-08-kfserving-0.5.md (hunk 2 above)
Comment (animeshsingh) [reply _posts/2021-03-08-kfserving-0.5.md:33]: can you add as suggestion to commit @yuzisun ?

File: _posts/2021-03-08-kfserving-0.5.md [hunk 4]
```diff
@@ -29,9 +29,9 @@ KFServing 0.5 also introduces an optional model agent for request/response loggi
...
 
--   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>bert</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>torchscript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
+-   Triton Inference Server V2 inference REST/gRPC protocol support, see examples of serving [<u>BERT</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/bert) and [<u>TorchScript</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/triton/torchscript) models on GPUs.
 
--   Tensorflow <u>gRPC support</u>.
+-   Tensorflow [<u>gRPC support</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/tensorflow#create-the-inferenceservice-with-grpc).
 
 -   [<u>SKLearn</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/sklearn)/<u>XGBoost</u> model server now uses [<u>MLServer</u>](https://github.com/SeldonIO/MLServer) which supports v2 inference protocol.
```
Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:36]:
```suggestion
-   [<u>SKLearn</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/sklearn)/[<u>XGBoost</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/xgboost) model server now uses [<u>MLServer</u>](https://github.com/SeldonIO/MLServer) which supports v2 inference protocol.
```

Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:36]:
```suggestion
-   [<u>SKLearn</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/sklearn)/[<u>XGBoost</u>](https://github.com/kubeflow/kfserving/tree/master/docs/samples/v1beta1/xgboost) model server now uses [<u>MLServer</u>](https://github.com/SeldonIO/MLServer) which supports v2 inference protocol.
```

File: _posts/2021-03-08-kfserving-0.5.md [hunk 5]
```diff
@@ -87,3 +87,61 @@ Please join us on the [<u>KFServing GitHub repository</u>](https://github.com/k
...
+
+We'd like to thank all the KFServing contributors for the awesome work!
+
+-   [Animesh Singh](https://github.com/animeshsingh)
+
+-   [Jinchi He](https://github.com/jinchihe) 
+
+-   [Clive Cox](https://github.com/cliveseldon)
+
+-   [Ellis Tarn](https://github.com/ellistarn)[ ]{.ul}
```
Comment (yuzisun) [inline _posts/2021-03-08-kfserving-0.5.md:101]: are these `[ ]{.ul}` needed?

File: _posts/2021-03-08-kfserving-0.5.md (hunk 2 above)
Comment (yuzisun) [reply _posts/2021-03-08-kfserving-0.5.md:33]: @animeshsingh added in suggestion. please help update

File: _posts/2021-03-08-kfserving-0.5.md (hunk 5 above)
Comment (animeshsingh) [reply _posts/2021-03-08-kfserving-0.5.md]: removed

File: _posts/2021-03-08-kfserving-0.5.md (hunk 2 above)
Comment (animeshsingh) [reply _posts/2021-03-08-kfserving-0.5.md:33]: done

**Comment by animeshsingh** [comment]
@james-jwu @theadactyl @kramachandran can one of you merge this?

**Pull request merged**